Index: muon-5.2.2-1/libmuonapt/CMakeLists.txt
===================================================================
--- muon-5.2.2-1.orig/libmuonapt/CMakeLists.txt
+++ muon-5.2.2-1/libmuonapt/CMakeLists.txt
@@ -6,9 +6,12 @@ set(muonapt_SRCS
     HistoryView/HistoryView.cpp
 )
 
+find_package(KF5 REQUIRED KDELibs4Support)
+
+
 add_library(MuonApt ${muonapt_SRCS})
 target_link_libraries(MuonApt Qt5::Core
-     KF5::Solid KF5::KIOWidgets KF5::XmlGui
+     KF5::Solid KF5::KIOWidgets KF5::XmlGui KF5::KDELibs4Support
      QApt::Main muonprivate
 )
 
