commit 027818bd078781bfd8eb4a4dee136baf61b40d1e
Author: Jonathan Riddell <jr@jriddell.org>
Date:   Thu Apr 2 12:04:28 2015 +0200

    fix install location of notifyrc files

diff --git a/libmuon/CMakeLists.txt b/libmuon/CMakeLists.txt
index 4f19c94..26cd0b8 100644
--- a/libmuon/CMakeLists.txt
+++ b/libmuon/CMakeLists.txt
@@ -62,4 +62,4 @@ target_include_directories(muonprivate PRIVATE ${PHONON_INCLUDES} PUBLIC ${CMAKE
 
 install(TARGETS muonprivate ${INSTALL_TARGETS_DEFAULT_ARGS})
 install(FILES moo.ogg DESTINATION ${DATA_INSTALL_DIR}/libmuon)
-install(FILES resources/muonabstractnotifier.notifyrc DESTINATION ${DATA_INSTALL_DIR}/muonabstractnotifier)
+install(FILES resources/muonabstractnotifier.notifyrc DESTINATION ${KNOTIFYRC_INSTALL_DIR})
diff --git a/libmuon/backends/ApplicationBackend/CMakeLists.txt b/libmuon/backends/ApplicationBackend/CMakeLists.txt
index 00d1b3e..a1c57c4 100644
--- a/libmuon/backends/ApplicationBackend/CMakeLists.txt
+++ b/libmuon/backends/ApplicationBackend/CMakeLists.txt
@@ -37,4 +37,4 @@ add_library(MuonApplicationNotifier MODULE ApplicationNotifier.cpp)
 target_link_libraries(MuonApplicationNotifier KF5::CoreAddons KF5::I18n KF5::Notifications KF5::IconThemes MuonNotifiers)
 
 install(TARGETS MuonApplicationNotifier DESTINATION ${PLUGIN_INSTALL_DIR}/muon-notifier)
-install(FILES muonapplicationnotifier.notifyrc DESTINATION ${DATA_INSTALL_DIR}/muonapplicationnotifier)
+install(FILES muonapplicationnotifier.notifyrc DESTINATION ${KNOTIFYRC_INSTALL_DIR})
