From a6c1b076931c402483dda872066bfb62cfada5a8 Mon Sep 17 00:00:00 2001
From: Jonathan Riddell <jr@jriddell.org>
Date: Wed, 25 Mar 2015 13:11:38 +0100
Subject: [PATCH] check for KDELibs4Support

---
 libmuonapt/CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/libmuonapt/CMakeLists.txt b/libmuonapt/CMakeLists.txt
index 1be9d70..69a8ea3 100644
--- a/libmuonapt/CMakeLists.txt
+++ b/libmuonapt/CMakeLists.txt
@@ -6,9 +6,12 @@ set(muonapt_SRCS
     HistoryView/HistoryView.cpp
 )
 
+find_package(KF5 REQUIRED KDELibs4Support)
+
+
 add_library(MuonApt ${muonapt_SRCS})
 target_link_libraries(MuonApt Qt5::Core
-     KF5::Solid KF5::KIOWidgets KF5::XmlGui
+     KF5::Solid KF5::KIOWidgets KF5::XmlGui KF5::KDELibs4Support
      QApt::Main muonprivate
 )
 
-- 
2.1.4

